package.json

```diff
@@ -20,7 +20,8 @@
     "test": "karma start --single-run"
   },
   "peerDependencies": {
-    "react": ">=15 || ^0.14.7"
+    "prop-types": "^15.5.8",
```

Comment:
Peer dependencies should be kept as loose as possible to make it easy for people using different versions. Let's use `>=15` here, same as `react`.